gwctl/pkg/printer/policies.go

```diff
@@ -40,20 +43,24 @@ func (pp *PoliciesPrinter) Print(policies []policymanager.Policy) {
 	})
 
 	tw := tabwriter.NewWriter(pp.Out, 0, 0, 2, ' ', 0)
-	row := []string{"POLICY NAME", "POLICY KIND", "TARGET NAME", "TARGET KIND", "POLICY TYPE"}
+	row := []string{"NAME", "KIND", "TARGET NAME", "TARGET KIND", "POLICY TYPE", "AGE"}
```

Comment:
Lets also configure the `KIND` column to output the resource type in the format `<KIND>.<GROUP>`. Example: `timeoutpolicy.foo.com`